package/Parser.roc

```diff
@@ -25,9 +25,9 @@
 ##
 ## parseGame : Str -> Result Game [ParsingError]
 ## parseGame = \s ->
```

Comment:
this header needs to finish getting converted to PNC and snake_case, both are missing